posthog/models/filters/mixins/property.py

```diff
@@ -22,14 +22,17 @@ def properties(self) -> List[Property]:
         else:
             loaded_props = _props
 
-        return self._parse_properties(loaded_props)
+        # if grouped properties
+        if isinstance(loaded_props, dict) and "type" in loaded_props and "groups" in loaded_props:
```

Comment:
We have two divergent paths here. IMO the saner way is to change `.properties` and not introduce `.property_groups` at all (we can always rename later if needed) but if we _really_ need both one should directly depend on the other.

Reply:
Ping/thoughts on this? I'd implement this in terms of `.property_groups` based on our call.

Reply:
Sorry, what's being asked here? I thought the plan now is to transition properties -> property_groups so it is a one way dependence. Also, every other property filtering area should remain the same until we make those changes. We're not changing entity properties to use or filtering yet so it can remain the same with the list of properties as long as we combine them with the property_group correctly during query building